core/variant/callable_bind.cpp

```diff
@@ -133,7 +133,7 @@ void CallableCustomBind::get_bound_arguments(Vector<Variant> &r_arguments, int &
 }
 
 void CallableCustomBind::call(const Variant **p_arguments, int p_argcount, Variant &r_return_value, Callable::CallError &r_call_error) const {
-	const Variant **args = (const Variant **)alloca(sizeof(const Variant **) * (binds.size() + p_argcount));
+	const Variant **args = (const Variant **)alloca(sizeof(const Variant *) * (binds.size() + p_argcount));
```

Comment:
```suggestion
	const Variant **args = (const Variant **)alloca(sizeof(Variant *) * (binds.size() + p_argcount));
```
Minor suggestion to match the other instances, such as `Object::callv`

Reply:
Did you want it part of this request or other one?

Reply:
As this, were already fixing it 🙂